R/alarm_add_plan.R

```diff
@@ -0,0 +1,23 @@
+#' This function facilitates comparing an existing (i.e., non-simulated) redistricting plan to a set of simulated plans.
```

Comment:
Remove "This function" at the start. This is a good description line.

You will also need a line above that is the actual function title (the header on the help page), followed by a blank `#'` line